geode/dispatcher.py

```diff
@@ -92,9 +92,10 @@ def __init__(self, config=None):
         idx = create_dist_index(origins, destinations)
 
         # kick off cache request
-        cache_future = asyncio.ensure_future(
-            self.cache.get_distances(
-                origins, destinations, provider=provider))
+        if self.cache:
```

Comment:
making cache optional (mainly for testing)

Reply:
constructor

Reply:
I often appreciate your brevity.